Makefile.am

```diff
@@ -102,7 +128,7 @@ rpm: rpm.spec
 	git clone "$$SrcDir" "mstpd-$$Version" ; \
 	tar -cvjf "mstpd-$$Version.tar.bz2" "mstpd-$$Version" ; \
 	rm -rf "mstpd-$$Version" ; \
-	cd .. ; mkdir -p SPECS ; cp "$$SrcDir/rpm.spec" SPECS/mstpd.spec
+	cd .. ; mkdir -p SPECS ; cp "$$SrcDir/rpm.spec" SPECS/mstpd.spec; \
```

Comment:
This will cause a merge conflict with #119 ... You might want to skip it here to avoid that.

Reply:
Cool... was just fixing bug in the file (and perhaps 119 wouldn't be merged :), reverted the fix.